src/cartesian/ErrorBar.tsx

```diff
@@ -102,12 +97,12 @@ function ErrorBarImpl(props: Props) {
   }
 
   // ErrorBar requires type number XAxis, why?
-  if (props.direction === 'x' && xAxis.type !== 'number') {
+  if (direction === 'x' && xAxis.type !== 'number') {
```

Comment:
So this is a breaking change where the user now specifies X or Y instead of layout?

Reply:
Layout is injected through element cloning, it's not an external prop. Also it entirely depends on direction anyway.